dev/wlp-gradle/subprojects/fat.gradle

```diff
@@ -152,7 +152,7 @@ task autoFVT {
   dependsOn ':cnf:copyMavenLibs'
   dependsOn addRequiredLibraries
   dependsOn copyFeatureBundles
-  dependsOn ':fattest.simplicity:assemble'
+  dependsOn ':fattest.simplicity:assembleBinaryDependencies'
```

Comment:
Can you explain this change?

Reply:
That's a change pulled in from integration that was part of Jared's update to fattest.simplicity.

Reply:
let's rebase :)